types/genesis.go

```diff
@@ -47,7 +47,7 @@ func (genDoc *GenesisDoc) SaveAs(file string) error {
 	if err != nil {
 		return err
 	}
-	return cmn.WriteFile(file, genDocBytes, 0644)
+	return cmn.WriteFile(file, cmn.MustSortJSON(genDocBytes), 0644)
```

Comment:
I think we should make this explicit by adding `sort bool` parameter.

Reply:
Also, "SaveAs is a utility method for saving GenensisDoc as a JSON file." comment needs to be updated